src/mappings/batchTransactionsHandler.ts

```diff
@@ -31,6 +31,8 @@ export async function batchTransactionsHandler(extrinsic: SubstrateExtrinsic): P
 
     logger.debug("Caught batch transaction extrinsic")
 
+    // TODO Depending on the result of the execution, divide the logic of the stored data (just like in other handlers)
```

Comment:
todo is here

Reply:
This improvement requires quite significant amount of time. I will remove the comment for now.